pkg/controller/clustersync/clustersync_controller.go

```diff
@@ -88,13 +88,22 @@ var (
 		},
 		[]string{"type", "result"},
 	)
+
+	metricTimeToApplySyncSets = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "hive_clustersync_time_to_apply_all_syncsets_seconds",
+			Help:    "Time between cluster install complete and all syncsets applied.",
```

Comment:
Suggest: Time from cluster install/adoption to first successful apply of all matching SyncSets and SelectorSyncSets.